libs/common/utilities/src/css/layout/padding/stories/vl-padding.stories.ts

```diff
@@ -0,0 +1,90 @@
+import { css } from 'lit';
+import { html } from 'lit-html';
+import vlPaddingStoriesDoc from './vl-padding.stories-doc.mdx';
+
+export default {
+    id: 'styles-next-layout-afnemers-padding',
+    title: 'Styles-next/Layout (afnemers)/padding',
+    tags: ['autodocs'],
+    parameters: {
+        docs: {
+            page: vlPaddingStoriesDoc,
+        },
+    },
+};
+
+const paddingCss = css`
+    .block {
+        width: 300px;
+        height: 100px;
+        background-color: lightblue;
+        padding: var(--vl-spacing--medium) 0; /* wordt via vl-region gezet */
```

Comment:
Dit moet `vl-section` worden.

Ik vind dit ook een beetje verwarrend. Ik zou het dan eerder in combinatie met `vl-section` gebruiken, ipv een andere klasse die hetzelfde doet als `vl-section`?

Ik stel voor eventueel de gerelateerde code van `vl-section` te plaatsen die de `padding` wijzigt (met dan bv. commentaar `/* deze css moet je zelf niet plaatsen */`. Echter, zoals ik een eerdere commentaar al vermelde; ik vind het beter dat de code voorbeelden, zeker onder sectie `Afnemen` focust op de essentiële informatie om de werking te begrijpen.

bv. in de `vl-section` documentatie zijn de code voorbeelden bv. al heel wat duidelijker

Reply:
aangepast: code via de 'parameters' versimpelt, kleuren / borders aangepast, tekst toegevoegd